### PR TITLE
Support UTF-8 content in scripts

### PR DESCRIPTION
**Pull request by abullrd**

### CODE REVIEW

File: tasks/mocha_jsdom.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = function(grunt) {
             .concat(makeScriptData('(' + mochaRunner.toString() + '())'));
 
         var env = {
-            html: '<link rel="stylesheet" href="node_modules/mocha/mocha.css"><div id="mocha"/>',
+            html: '<meta http-equiv="content-type" content="text/html; charset=UTF8"><link rel="stylesheet" href="node_modules/mocha/mocha.css"><div id="mocha"/>',
```
Comment (crteal) [inline tasks/mocha_jsdom.js]: Does `<meta charset="utf-8">` work?

**Comment by crteal** [comment]
Thanks for tracking this down, and for figuring it out! 👍

**Comment by crteal** [comment]
We'll have to push this to NPM afterwards, too.

**Comment by abullrd** [comment]
thanks

**Pull request merged**